xml/ha_troubleshooting.xml

```diff
@@ -129,8 +129,14 @@
     <term>Where to find the log files? </term>
     <listitem>
      <para>
+      <remark>taroth 2018-03-05: ygao, should I replace the following with
+     "Pacemaker writes its logs to '/var/log/pacemaker/pacemaker.log' and
+     '/var/log/pacemaker/bundles/*/pacemaker.log, respectively"?
+     (whatever the bundle logs are...), see c#4</remark>
       For the Pacemaker log files, see the settings configured in the
-      <literal>logging</literal> section of &corosync.conf;. In case the
+      <literal>logging</literal> section of &corosync.conf;. <remark>taroth
+      2018-03-09: ygao, the logging section in /etc/sysconfig/pacemaker still
+      exists, should we therefore keep the following sentence? </remark>In case the
```

Comment:
Yes, and /etc/sysconfig/pacemaker is the only  place for pacemaker log settings now.